script.js

```diff
@@ -9,3 +9,26 @@
 // Add a `facial-hair` property to `instructor`.
 
 // Log the value of the `facial-hair` property to the console.
+
+
+let colors= ["red", "yellow", "green"]
+
+
+let green = colors[2]
+
+for (index = 0; index < colors.length; ++index){
+  console.log("my favorite color is")
+}
+// colors.forEach(fucntion(find)){
+  //console.log("my favorite color is .find")
```

Comment:
could do this like: <code>colors.forEach((color) => console.log(\`my favorite color is ${color}\`))</code>